### ISSUE TITLE
RuntimeError: File at path ...\site-packages\gradio\templates\frontend\assets/js/GlobalStyles.c3df4297.js does not exist.

### ISSUE THREAD

**Issue by off99555** [gradio-app/gradio#2986]
### Describe the bug

I run simple getting started code and I got error missing files when I load the webpage. Every time I run the code I got different file missing errors.

### Is there an existing issue for this?
No
- [X] I have searched the existing issues

### Reproduction

Run this code using latest version of `gradio` (3.16.1):
```py
import gradio as gr

def greet(name):
    return "Hello " + name + "!"

demo = gr.Interface(fn=greet, inputs="text", outputs="text")

demo.launch()   
```

### Screenshot

![image](https://user-images.githubusercontent.com/15215732/212564709-6ee26742-0dee-4fac-8911-5acc069f3549.png)

### Logs
Don't be confused about the format of the traceback because I use this package: https://github.com/Qix-/better-exceptions

```shell
ERROR:    Exception in ASGI application
Traceback (most recent call last):
  File "C:\Users\off99\anaconda3\lib\site-packages\starlette\responses.py", line 282, in __call__
    stat_result = await anyio.to_thread.run_sync(os.stat, self.path)
                        |                        |        -> <starlette.responses.FileResponse object at 0x000001C12539EF40>
                        |                        -> <module 'os' from 'C:\\Users\\off99\\anaconda3\\lib\\os.py'>
                        -> <module 'anyio' from 'C:\\Users\\off99\\AppData\\Roaming\\Python\\Python38\\site-packages\\anyio\\__init__.py'>
  File "C:\Users\off99\AppData\Roaming\Python\Python38\site-packages\anyio\to_thread.py", line 28, in run_sync
    return await get_asynclib().run_sync_in_worker_thread(func, *args, cancellable=cancellable,
  File "C:\Users\off99\AppData\Roaming\Python\Python38\site-packages\anyio\_backends\_asyncio.py", line 802, in run_sync_in_worker_thread
    return await future
                 -> <Future finished exception=FileNotFoundError(2, 'The system cannot find the path specified')>
  File "C:\Users\off99\AppData\Roaming\Python\Python38\site-packages\anyio\_backends\_asyncio.py", line 740, in run
    result = func(*args)
    |        |     -> ('C:\\Users\\off99\\anaconda3\\lib\\site-packages\\gradio\\templates\\frontend\\assets/js/GlobalStyles.c3df4297.js',)
    |        -> <built-in function stat>
    -> None
FileNotFoundError: [WinError 3] The system cannot find the path specified: 'C:\\Users\\off99\\anaconda3\\lib\\site-packages\\gradio\\templates\\frontend\\assets/js/GlobalStyles.c3df4297.js'
...
AND A LOT MORE ERRORS. I TRUNCATED BECAUSE IT'S TOO LONG FOR THE POST.
...
FileNotFoundError: [WinError 3] The system cannot find the path specified: 'C:\\Users\\off99\\anaconda3\\lib\\site-packages\\gradio\\templates\\frontend\\assets/js/GlobalStyles.c3df4297.js'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Users\off99\anaconda3\lib\site-packages\uvicorn\protocols\http\httptools_impl.py", line 375, in run_asgi
    result = await app(self.scope, self.receive, self.send)
                   |   |           |             -> <uvicorn.protocols.http.httptools_impl.RequestResponseCycle object at 0x0000022E3C6860D0>
                   |   |           -> <uvicorn.protocols.http.httptools_impl.RequestResponseCycle object at 0x0000022E3C6860D0>
                   |   -> <uvicorn.protocols.http.httptools_impl.RequestResponseCycle object at 0x0000022E3C6860D0>
                   -> <uvicorn.middleware.proxy_headers.ProxyHeadersMiddleware object at 0x0000022E3C553490>
  File "C:\Users\off99\anaconda3\lib\site-packages\uvicorn\middleware\proxy_headers.py", line 75, in __call__
    return await self.app(scope, receive, send)
                 |        |      |        -> <bound method RequestResponseCycle.send of <uvicorn.protocols.http.httptools_impl.RequestResponseCycle object at 0x0000022E3C686...
                 |        |      -> <bound method RequestResponseCycle.receive of <uvicorn.protocols.http.httptools_impl.RequestResponseCycle object at 0x0000022E3C...
                 |        -> {'type': 'http', 'asgi': {'version': '3.0', 'spec_version': '2.1'}, 'http_version': '1.1', 'server': ('127.0.0.1', 7860), 'clien...
                 -> <uvicorn.middleware.proxy_headers.ProxyHeadersMiddleware object at 0x0000022E3C553490>
  File "C:\Users\off99\anaconda3\lib\site-packages\fastapi\applications.py", line 208, in __call__
    await super().__call__(scope, receive, send)
                           |      |        -> <bound method RequestResponseCycle.send of <uvicorn.protocols.http.httptools_impl.RequestResponseCycle object at 0x0000022E3C686...
                           |      -> <bound method RequestResponseCycle.receive of <uvicorn.protocols.http.httptools_impl.RequestResponseCycle object at 0x0000022E3C...
                           -> {'type': 'http', 'asgi': {'version': '3.0', 'spec_version': '2.1'}, 'http_version': '1.1', 'server': ('127.0.0.1', 7860), 'clien...
  File "C:\Users\off99\anaconda3\lib\site-packages\starlette\applications.py", line 112, in __call__
    await self.middleware_stack(scope, receive, send)
          |                     |      |        -> <bound method RequestResponseCycle.send of <uvicorn.protocols.http.httptools_impl.RequestResponseCycle object at 0x0000022E3C686...
          |                     |      -> <bound method RequestResponseCycle.receive of <uvicorn.protocols.http.httptools_impl.RequestResponseCycle object at 0x0000022E3C...
          |                     -> {'type': 'http', 'asgi': {'version': '3.0', 'spec_version': '2.1'}, 'http_version': '1.1', 'server': ('127.0.0.1', 7860), 'clien...
          -> <gradio.routes.App object at 0x0000022E3BA04F40>
  File "C:\Users\off99\anaconda3\lib\site-packages\starlette\middleware\errors.py", line 181, in __call__
    raise exc
  File "C:\Users\off99\anaconda3\lib\site-packages\starlette\middleware\errors.py", line 159, in __call__
    await self.app(scope, receive, _send)
          |        |      |        -> <function ServerErrorMiddleware.__call__.<locals>._send at 0x0000022E3C7E83A0>
          |        |      -> <bound method RequestResponseCycle.receive of <uvicorn.protocols.http.httptools_impl.RequestResponseCycle object at 0x0000022E3C...
          |        -> {'type': 'http', 'asgi': {'version': '3.0', 'spec_version': '2.1'}, 'http_version': '1.1', 'server': ('127.0.0.1', 7860), 'clien...
          -> <starlette.middleware.errors.ServerErrorMiddleware object at 0x0000022E3C542040>
  File "C:\Users\off99\anaconda3\lib\site-packages\starlette\middleware\cors.py", line 92, in __call__
    await self.simple_response(scope, receive, send, request_headers=headers)
          |                    |      |        |                     -> Headers({'host': '127.0.0.1:7860', 'connection': 'keep-alive', 'sec-ch-ua': '"Not_A Brand";v="99", "Google Chrome";v="109", "Chr...
          |                    |      |        -> <function ServerErrorMiddleware.__call__.<locals>._send at 0x0000022E3C7E83A0>
          |                    |      -> <bound method RequestResponseCycle.receive of <uvicorn.protocols.http.httptools_impl.RequestResponseCycle object at 0x0000022E3C...
          |                    -> {'type': 'http', 'asgi': {'version': '3.0', 'spec_version': '2.1'}, 'http_version': '1.1', 'server': ('127.0.0.1', 7860), 'clien...
          -> <starlette.middleware.cors.CORSMiddleware object at 0x0000022E3C535EE0>
  File "C:\Users\off99\anaconda3\lib\site-packages\starlette\middleware\cors.py", line 147, in simple_response
    await self.app(scope, receive, send)
          |        |      |        -> functools.partial(<bound method CORSMiddleware.send of <starlette.middleware.cors.CORSMiddleware object at 0x0000022E3C535EE0>>,...
          |        |      -> <bound method RequestResponseCycle.receive of <uvicorn.protocols.http.httptools_impl.RequestResponseCycle object at 0x0000022E3C...
          |        -> {'type': 'http', 'asgi': {'version': '3.0', 'spec_version': '2.1'}, 'http_version': '1.1', 'server': ('127.0.0.1', 7860), 'clien...
          -> <starlette.middleware.cors.CORSMiddleware object at 0x0000022E3C535EE0>
  File "C:\Users\off99\anaconda3\lib\site-packages\starlette\exceptions.py", line 82, in __call__
    raise exc
  File "C:\Users\off99\anaconda3\lib\site-packages\starlette\exceptions.py", line 71, in __call__
    await self.app(scope, receive, sender)
          |        |      |        -> <function ExceptionMiddleware.__call__.<locals>.sender at 0x0000022E3C7E8430>
          |        |      -> <bound method RequestResponseCycle.receive of <uvicorn.protocols.http.httptools_impl.RequestResponseCycle object at 0x0000022E3C...
          |        -> {'type': 'http', 'asgi': {'version': '3.0', 'spec_version': '2.1'}, 'http_version': '1.1', 'server': ('127.0.0.1', 7860), 'clien...
          -> <starlette.exceptions.ExceptionMiddleware object at 0x0000022E3C535FD0>
  File "C:\Users\off99\anaconda3\lib\site-packages\starlette\routing.py", line 589, in __call__
    await route.handle(scope, receive, send)
          |            |      |        -> <function ExceptionMiddleware.__call__.<locals>.sender at 0x0000022E3C7E8430>
          |            |      -> <bound method RequestResponseCycle.receive of <uvicorn.protocols.http.httptools_impl.RequestResponseCycle object at 0x0000022E3C...
          |            -> {'type': 'http', 'asgi': {'version': '3.0', 'spec_version': '2.1'}, 'http_version': '1.1', 'server': ('127.0.0.1', 7860), 'clien...
          -> <fastapi.routing.APIRoute object at 0x0000022E3C56E070>
  File "C:\Users\off99\anaconda3\lib\site-packages\starlette\routing.py", line 250, in handle
    await self.app(scope, receive, send)
          |        |      |        -> <function ExceptionMiddleware.__call__.<locals>.sender at 0x0000022E3C7E8430>
          |        |      -> <bound method RequestResponseCycle.receive of <uvicorn.protocols.http.httptools_impl.RequestResponseCycle object at 0x0000022E3C...
          |        -> {'type': 'http', 'asgi': {'version': '3.0', 'spec_version': '2.1'}, 'http_version': '1.1', 'server': ('127.0.0.1', 7860), 'clien...
          -> <fastapi.routing.APIRoute object at 0x0000022E3C56E070>
  File "C:\Users\off99\anaconda3\lib\site-packages\starlette\routing.py", line 55, in app
    await response(scope, receive, send)
          |        |      |        -> <function ExceptionMiddleware.__call__.<locals>.sender at 0x0000022E3C7E8430>
          |        |      -> <bound method RequestResponseCycle.receive of <uvicorn.protocols.http.httptools_impl.RequestResponseCycle object at 0x0000022E3C...
          |        -> {'type': 'http', 'asgi': {'version': '3.0', 'spec_version': '2.1'}, 'http_version': '1.1', 'server': ('127.0.0.1', 7860), 'clien...
          -> <starlette.responses.FileResponse object at 0x0000022E3C5EEA60>
  File "C:\Users\off99\anaconda3\lib\site-packages\starlette\responses.py", line 285, in __call__
    raise RuntimeError(f"File at path {self.path} does not exist.")
                                       -> <starlette.responses.FileResponse object at 0x0000022E3C5EEA60>
RuntimeError: File at path C:\Users\off99\anaconda3\lib\site-packages\gradio\templates\frontend\assets/js/GlobalStyles.c3df4297.js does not exist.
```

### System Info

```shell
>>> gradio.__version__
'3.16.1'
```

### Severity

annoying

**Comment by off99555** [comment]
> Hi @off99555 I cannot reproduce with either 3.16.1 or 3.16.2
> 
> <img alt="image" width="1271" src="https://user-images.githubusercontent.com/41651716/212674911-fe2920d4-aa46-440a-89f8-a800f4b76e53.png">
> 
> <img alt="image" width="470" src="https://user-images.githubusercontent.com/41651716/212674939-4de2f447-1c70-4389-b799-92f07fc35f73.png">

Try this colab notebook: https://colab.research.google.com/drive/152jNY_TWYqQdKOe2GTDZY50gXr8hHO3Q?usp=sharing
You can run all the cells and click on the following link:
![image](https://user-images.githubusercontent.com/15215732/212677566-87cf36cf-43ca-4dba-9a29-572c6643fce8.png)
Once the webpage opens, you should see errors similar to the following image:
![image](https://user-images.githubusercontent.com/15215732/212678034-78d87b42-99c8-4852-81cb-df399d02d469.png)

**Comment by freddyaboulton** [comment]
Hmm I do not see that @off99555 !

<img width="1446" alt="image" src="https://user-images.githubusercontent.com/41651716/212681156-7f6e7d87-17fa-48cb-abe5-f7eaef2a1804.png">


Thanks for the through reproducer steps though.

**Comment by off99555** [comment]
It's weird. I had this issue both when running it locally and on Colab. When I clicked on the localhost link, the error appears.
Maybe it's related to web browser. I normally use Chrome and it had that error.
I just tried it with Edge and there is no error. Do you have any clue why it would be related to the browser?

**Comment by freddyaboulton** [comment]
Hmm maybe Chrome cached a js file from a previous version or something like that. Not sure though!

**Comment by off99555** [comment]
OK. I tried restarting computer, and disabling a few chrome extensions, and I finally arrived at the cause. It's this [Nanobyte extension](https://chrome.google.com/webstore/detail/nanobyte/ndkdijcnlhjhmakblkhmpjocfjjifhbo) that is the problem.
I'm not sure why though. It's an extension for sending a cryptocurrency called Nano on websites. Maybe it's modifying something on the page.

Here is the article about this extension:
https://medium.com/@charlie_51843/introducing-nanobyte-333251a39fd3
https://nanobytepay.com/
